doc/api/source/index.html.md.erb

```diff
@@ -129,23 +164,13 @@ To get data points for each day in the requested time frame:
 curl -H 'Authorization: Bearer your-token-here' https://content-performance-manager.publishing.service.gov.uk/api/v1/metrics/time-series/f838c22a-b2aa-49be-bd95-153f593293a3?from=2018-02-28&to=2018-02-28&metric=pageviews
 ```
 
-## Version
+## API reference
 
-GOV.UK Content Performance API is currently alpha software and is likely to
-change significantly between versions.
+All examples are written in curl.
```

Comment:
I would move this below the statement below

Reply:
Done